src/apps/services/credits.py

```diff
@@ -96,11 +95,7 @@ def vintage_agg(self, df):
 
     @chained_cached_command()
     def countries_agg(self, df):
-        df["country_code"] = [
-            Countries().get_country(country) for country in df["country"]
-        ]
-        df["country_text"] = df["country_code"].astype(str)
-        df = df.groupby(["country", "country_text", "country_code"])
+        df = df.groupby(["country", "country_code"])
```

Comment:
are we handling this on the frontend instead?

Reply:
Yes, We may need to add other columns to this dataset but the formatting will be done in the UI (we do not know yet which framework we are going to use to render the map)